<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDlgQueryOwner</class>
 <widget class="QDialog" name="CDlgQueryOwner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>查询业主资料</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(D:/WinParkUI/debug/Image/NewIcon/CommonBG-normal.jpg);</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>120</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(none);</string>
   </property>
   <property name="text">
    <string>车牌号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>150</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(none);</string>
   </property>
   <property name="text">
    <string>业主姓名：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edt2q">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="statusTip">
    <string> and c.cardno like '%%1%'</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(none);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edt1q">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>61</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="statusTip">
    <string> and c.cardselfno like '%%1%'</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(none);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>90</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(none);</string>
   </property>
   <property name="text">
    <string>卡号：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edt4q">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="statusTip">
    <string> and a.username like '%%1%'</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(none);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>61</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(none);</string>
   </property>
   <property name="text">
    <string>自编号：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>337</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(none);</string>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="edt5q">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>271</width>
     <height>64</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(none);</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>180</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(none);</string>
   </property>
   <property name="text">
    <string>备注：</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblTitle">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>5</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(none);
color: rgb(85, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnQuery">
   <property name="geometry">
    <rect>
     <x>256</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(none);</string>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edt3q">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="statusTip">
    <string> and b.carcp like '%%1%'</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(none);</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblClose">
   <property name="geometry">
    <rect>
     <x>423</x>
     <y>6</y>
     <width>30</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
